README.md

```diff
@@ -63,12 +63,12 @@ Databend uses the latest techniques in vectorized query processing to allow you
 
 - __Blazing Performance__
 
-  Databend leverages data-level parallelism(Vectorized Query Execution) and instruction-level parallelism(SIMD) technology, offering blazing performance data analytics.
+  Databend leverages data-level parallelism(Vectorized Query Execution) and instruction-level parallelism(SIMD) technology, offering [blazing performance](https://benchmark.clickhouse.com/) data analytics.
 
 
 - __Git-like MVCC Storage__
 
-  Databend stores data with snapshots. It's easy to query, clone, and restore historical data in tables.
+  [Databend stores data with snapshots](https://databend.rs/blog/databend-engine). Databend can flash back to any history timepoint of data, it's easy to query, clone, and restore historical data in tables.
```

Comment:
Can we use this link in the sentence? https://databend.rs/doc/sql-commands/ddl/table/optimize-table#what-are-snapshot-segment-and-block

And i updated the sentence a little bit:

[Databend uses snapshots to store data](https://databend.rs/doc/sql-commands/ddl/table/optimize-table#what-are-snapshot-segment-and-block), enabling users to effortlessly query, clone, or restore data from any history timepoint.